package.json

```diff
@@ -29,11 +29,11 @@
     "express-query-int": "^3.0.0"
   },
   "peerDependencies": {
-    "bedrock": "1.12.1 - 3.x",
-    "bedrock-account": "^2.0.0",
-    "bedrock-express": "2.0.8 - 3.x",
+    "bedrock": "2.x - 3.x",
+    "bedrock-account": "2.x - 3.x",
+    "bedrock-express": "2.x - 3.x",
     "bedrock-passport": "4.x - 5.x",
-    "bedrock-validation": "^4.1.0"
+    "bedrock-validation": "3.x - 4.x"
```

Comment:
I actually don't know if this is true. @mattcollier @davidlehn  is this lib backwards compatible with `bedrock-validation` 2 and 3?

Reply:
I don't think we want to make this change.  Maybe it was a typo?

Reply:
so we want `bedrock-validation": ^4.0.0`?

Reply:
I wasn't sure. In the issue description, it was mentioned that the peerDepedencies will need to follow this pattern lastRelease.x - latestRelease.x, how can we tell if a lib is backwards compatible?